src/coreclr/vm/fcall.h

```diff
@@ -785,40 +263,8 @@ void HCallAssert(void*& cache, void* target);
 // Macros that allows fcall to be split into two function to avoid the helper frame overhead on common fast
```

Comment:
Everything from `FC_INNER` till `FC_INNER_RETURN_VOID` should be deleted.

Also, all instances of `FC_NO_TAILCALL` should be deleted.